.github/scripts/tag_current_year_model_runs.R

```diff
@@ -0,0 +1,63 @@
+# Script to tag a list of model runs by ID from AWS.
+#
+# Accepts two arguments, a comma-delimited list of run IDs for model runs and
+# a new run_type which to apply to those runs. See delete script for more
+# details on the script implementation
```

Comment:
**[Thought, non-blocking]** There are probably some opportunities to factor out shared logic with the delete script, particularly in the date and run ID validation logic, but there are enough annoying little differences that I'm fine keeping them separate for now and revisiting later if we get to a point where we're having to make changes and keep both scripts in sync.